packages/block-library/src/breadcrumbs/block.json

```diff
@@ -8,10 +8,10 @@
 	"description": "Display a breadcrumb trail for hierarchical post types or based on taxonomy terms.",
 	"textdomain": "default",
 	"attributes": {
-		"type": {
+		"postBreadcrumbsType": {
```

Comment:
1. Don't we need a migration?
2. Since we are changing the attribute, why not a better name and/or better semantics? Focusing on "type" is a bit vague. Are you choosing that in order to allow some margin for new unforeseen types in the future? If not, what would be the downsides of a more literal option, closer to a simple `{boolean} [prefersTaxonomy=false]`?